codecarbon/external/hardware.py

```diff
@@ -82,6 +88,12 @@ def _get_power_from_cpus(self) -> Power:
         Get CPU power from Intel Power Gadget
         :return: power in kW
         """
+        if self._mode == "constant":
+            logger.warning(
+                "CODECARBON : No CPU/GPU tracking mode found. Falling back on CPU constant mode."
```

Comment:
Actually this could be triggered with GPU but no CPU

Reply:
Also, I'm not sure we should warn all the time. Thoughts @nikolaskaris @LiamConnell @Lothiraldan @vl4py @benoit-cty ?

Reply:
This is already able to be triggered with GPU but no CPU - I'll modify the warning message